tests/baselines/reference/privateNamesAndGenericClasses-2.errors.txt

```diff
@@ -28,6 +30,9 @@ tests/cases/conformance/classes/members/privateNames/privateNamesAndGenericClass
     let b = new C("hello");
     
     a.baz = 5                                 // OK
+    ~~~~~
+!!! error TS2322: Type 'number' is not assignable to type 'T'.

```

Comment:
this seems bad; maybe the ad-hoc type is missing in some instantiation step?

Reply:
New commit does instantiation (properly, I think...)